lib/Dialect/CGGI/IR/CGGIPBSOps.td

```diff
@@ -185,6 +185,47 @@ def CGGI_MultiLutLinCombOp : CGGI_Op<"multi_lut_lincomb", [
   let hasVerifier = 1;
 }
 
+def CGGI_SelectorLutLinCombOp : CGGI_Op<"selector_lut_lincomb", [
+  Pure,
+  ElementwiseMappable,
+  Scalarizable
+]> {
+  let summary = "A multi-value version of lut_lincomb with one LUT per output.";
```

Comment:
```suggestion
  let summary = "A multi-value version of lut_lincomb.";
```

maybe keep it at that? but i think i just don't understand what "one LUT per output" means (I interpreted that to mean that there will be n lookup_tables)